_config/session.yml

```
@@ -0,0 +1,39 @@
+---
+Name: session-handlers
+---
+SilverStripe\Core\Injector\Injector:
+  SilverStripe\Control\SessionHandler\CacheSessionHandler:
+    constructor:
+      cacheAdapter: '%$Psr\SimpleCache\CacheInterface.session-handler'
```

```suggestion
      cache: '%$Psr\SimpleCache\CacheInterface.session-handler'
```
This didn't actually match "cacheHandler", which I've recommended be changed to "cache"

---

Still needs to be done

---

Oops sorry, forgot about this one.
Done now.
Please don't forget to assign to me after reviewing :sweat_smile: